http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -273,15 +275,15 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
     for {
       traceId <- ev.ask
       _ <- IO.fromFuture(IO(validateClusterRequestBucketObjectUri(userEmail, googleProject, augmentedClusterRequest)))
-      _ <- IO(logger.info(s"[$traceId] Attempting to notify the AuthProvider for creation of cluster '$clusterName' " +
+      _ <- IO(logger.debug(s"[$traceId] Attempting to notify the AuthProvider for creation of cluster '$clusterName' " +
```

Comment:
I find the above and the below messages helpful when I look at logs and would prefer not changing the log level on them.

Reply:
I really think these are debug level logs. It's a bit distracting when reading code like this that we're basically logging after each action we do. I can see the value if we're actively debugging, but not as much if things are running normally.